lib/health_cards/health_card.rb

```diff
@@ -69,16 +69,26 @@ def fhir_version(ver = nil)
       end
 
       # Additional type claims this HealthCard class supports
-      # @param types [String, Array] A string or array of string representing the additional type claims
+      # @param types [String, Array] A string or array of string representing the additional type claims or nil
+      # if used as a getter
+      # @return [Array] the additional types added by this classes
       def additional_types(*add_types)
-        types.concat(add_types)
+        types.concat(add_types) unless add_types.nil?
+        types - VC_TYPE
       end
 
       # Type claims supported by this HealthCard subclass
       # @return [Array] an array of Strings with all the supported type claims
       def types
         @types ||= VC_TYPE.dup
       end
+
+      # Check if this class supports the given type
+      # @param type [String] A type as defined by the SMART Health Cards framework
+      # @return [Array] All additional types supported by this HealthCard (sub)class
```

Comment:
This needs updating.

Reply:
Do you mean the documentation or something else? Just want to make sure I'm not missing something

Reply:
Sorry, just the documentation.